image/templates/helm/stackrox-secured-cluster/templates/collector-scc.yaml

```diff
@@ -54,12 +10,12 @@ volumes:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: use-privileged-scc
```

Comment:
Renaming those might make the manual upgrade instructions a hell for you. Please keep it in mind.

Reply:
Good point. Let me roll that namechange back as much as possible.

Reply:
Rolled back where possible in https://github.com/stackrox/stackrox/pull/8663/commits/4b664b9d6c8213acfac453f44bf713eda1ffe002